Tests/ttLib/tables/_a_v_a_r_test.py

```diff
@@ -47,9 +47,7 @@ def test_decompile(self):
     def test_decompile_unsupportedVersion(self):
         avar = table__a_v_a_r()
         font = self.makeFont(["wdth", "wght"])
-        self.assertRaises(
-            TTLibError, avar.decompile, deHexStr("02 01 03 06 00 00 00 00"), font
-        )
+        TTLibError, avar.decompile, deHexStr("02 01 03 06 00 00 00 00"), font
```

Comment:
this line looks incomplete, where's the assert gone?

Reply:
they we should just remove the whole test I guess

Reply:
Just did that.